Source/Core/DolphinWX/Frame.h

```diff
@@ -145,6 +145,12 @@ class CFrame : public CRenderFrame
     ADD_PANE_CENTER
   };
 
+  enum class CycleDirection : int
+  {
+    FORWARD = 1,
```

Comment:
`enum class` members are CamelCased (preserving the case of any initialisms or abbreviations as well, if present). In this case, `Forward`, and `Backward`